assets/js/localization.js

```diff
@@ -207,18 +210,32 @@ function getLocales() {
         var localePairs = [];
         for(var code in locales) {
             localePairs.push([code, locales[code]]);
+            pageLocales[code] = locales[code];
         }
         localePairs = localePairs.sort(function (a, b) {
             return a[1].toLowerCase().localeCompare(b[1].toLowerCase());
         });
         $('.localeSelect').empty();
+        $('#localeDropdown').empty();
         $.each(localePairs, function () {
             $('.localeSelect').append(
                 $('<option></option>')
                     .val(this[0])
                     .text(this[1])
                     .prop('dir', rtlLanguages.indexOf(this[0]) !== -1 ? 'rtl' : 'ltr')
             );
+
+            $('#localeDropdown').append(
+                $('<li></li>').append(
+                    $('<a>', {
+                        href: 'index.' + this[0] + '.html'
```

Comment:
The current locale dropdown does not do this. We do not want to force a page refresh for switching languages! That's not great UX.